homeassistant/components/tuya/strings.json

```
@@ -237,6 +237,9 @@
       },
       "inverter_output_power_limit": {
         "name": "Inverter output power limit"
+      },
+      "fan_duration": {
+        "name": "Fan duration"
```

"fan duration" does not make clear whether this is counter or a countdown. As I understand it this is about the remaining runtime, so perhaps we should use that instead:

```suggestion
        "name": "Remaining fan runtime"
```

---

If it's a fixed value then "timer" sounds best!